content/rs/databases/active-active/manage-aa.md

```diff
@@ -0,0 +1,77 @@
+---
+Title: Manage Active-Active databases 
+description: Manage Active-Active
+weight: 30
+alwaysopen: false
+categories: ["RS"]
+aliases: [
+    /rs/databases/active-active/edit-aa-database.md,
+    /rs/databases/active-active/edit-aa-database/,
+    /rs/databases/active-active/manage-aa.md,
+    /rs/databases/active-active/manage-aa/,
+]
+---
+
+You can configure and manage your Active-Active database from either the admin console or the command line.
+
+When you edit the database configurations of an Active-Active database with the admin console or the `rladmin` CLI, the changes will only apply to the Active-Active database instance you are currently editing.
+
+To change the global configuration of the Active-Active database, use the [`crdb-cli`]({{<relref "/rs/references/cli-utilities/crdb-cli">}}).
+
+## Database settings
+
+The following table shows a list of database settings, tools you can use to change those settings, and links to more information.
+
+Much of the Active-Active database settings can be changed after the database has been created. One notable exception is database clustering. Database clustering can't be turned on or off after the database has been created and will remain the same through the lifetime of the database.
+
+## Participating clusters
+
+You can add and remove participating clusters of an Active-Active database to change the topology.
+Use the participating clusters list to manage the changes to Active-Active topology in the UI.
+You can make multiple changes at once and changes will be committed when the database configuration is saved.
+
+![add-active-active-participants](/images/rs/add-active-active-participants.png)
+
+### Add participating clusters
+
+All of the existing participating clusters must be online and in a syncing state when you add new participating clusters.
+
+New participating clusters create the Active-Active database instance based on the global Active-Active database configuration.
+After you add new participating clusters to an existing Active-Active database,
+the new database instance can accept connections and read operations.
+The new instance does not accept write operations until it is in the syncing state.
+
+### Remove participating clusters
+
+All of the existing participating clusters must be online and in a syncing state when you remove an online participating clusters.
+If you must remove offline participating clusters, you can do this with forced removal.
+If a participating cluster that was removed forcefully returns attempts to re-join the cluster,
+it will have an out of date on Active-Active database membership.
+The joined participating clusters reject updates sent from the removed participating cluster.
```

Comment:
We can add:
To avoid the re-join attempts, purge the forcefully removed instance from its participating cluster.